### PR TITLE
fix: task runs card's created by shows creator of task

### LINKED ISSUE #1864: Task Runs Card's 'created by' should show task creator's name
**Background** 
`Created by` section of the Task Runs Card component in the Task Runs page shows the wrong information. Instead of showing the name of the task, it should display the name of the task creator. 

**Acceptance Criteria** 

- created by section of the task runs card shows the creator of the task.

### PR DESCRIPTION
**Pull request by blegesse-git**
Closes #1864 

**Before** 
The `created by` section of task runs card shows the task name 

Screen Shot 2021-06-28 at 3 16 42 PM

**After**
`Created by` shows the creator of the task.

Screen Shot 2021-06-30 at 9 00 32 PM

### CODE REVIEW

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 59 | :red_circle:
accessibility | 96 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=master)

File: src/tasks/components/TaskRunsCard.tsx [hunk 1]
```diff
@@ -66,7 +74,7 @@ class UnconnectedTaskRunsCard extends PureComponent<
           <ResourceCard.Meta>
             {this.activeToggle}
             <>Created at: {task.createdAt}</>
```
Comment (hoorayimhelping) [inline src/tasks/components/TaskRunsCard.tsx:76]: let's take this opportunity to format this date

**Review by hoorayimhelping — COMMENTED** [review]
looks good! but let's format the date with our default date format in the task runs card

File: src/tasks/components/TaskRunsCard.tsx (hunk 1 above)
Comment (blegesse-git) [reply src/tasks/components/TaskRunsCard.tsx]: great idea!

**Review by hoorayimhelping — APPROVED** [review]

**Pull request merged**